server/lib/realtime/replication.ex

```diff
@@ -300,23 +356,6 @@ defmodule Realtime.Replication do
   end
 
   defp notify_subscribers(%State{transaction: {_current_txn_lsn, txn}}) do
-    txn
```

Comment:
This reverts https://github.com/w3b6x9/realtime/commit/e47148365ac89201a02c95745074d756ebfdd264. `commit_timestamp` is now added at the source when creating structs (i.e. `NewRecord`, `UpdatedRecord`, `DeletedRecord`, `TruncatedRelation`.